### ISSUE TITLE
Fast_Expression.c has magic numbers

### ISSUE THREAD

**Issue by jleeothon** [cortoproject/corto#319]
`Fast_Expression.c` has hardcoded constants for min and max numbers (e.g. MIN_INT8). We should replace them with constants in `cx__bootstrap.h`.

Should check the functions `Fast_Expression_getIntTypeFromValue` and `Fast_Expression_narrow`.

**Comment by SanderMertens** [comment]
What? =P

**Comment by SanderMertens** [comment]
I don't know what this is about, but if it's a style issue, it probably doesn't need to be solved for 0.2.0.

**Comment by SanderMertens** [comment]
This issue is no longer relevant as it exists in a different repository, and the `corto-language` repository will be deprecated once the `corto-parser` repository is ready.
